src/Mailer/Message/CitizenActionContactParticipantsMessage.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace AppBundle\Mailer\Message;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\EventRegistration;
+use Ramsey\Uuid\Uuid;
+
+final class CitizenActionContactParticipantsMessage extends Message
+{
+    /**
+     * @param EventRegistration[] $recipients
+     * @param Adherent            $organizer
+     * @param string              $content
+     *
+     * @return CitizenActionContactParticipantsMessage
+     */
+    public static function create(array $recipients, Adherent $organizer, string $content): self
+    {
+        $recipients = array_values($recipients);
```

Comment:
Can you add a comment to explain why is this needed ? It's to reset the key order ?